llm_bioinformatics_research/src/SignupScreen/SignupScreen.js

```diff
@@ -0,0 +1,247 @@
+import React, { useState, useEffect } from 'react';
+import { TextField, Button, Container, Typography, Box, IconButton, Link } from '@mui/material';
+import { Visibility, VisibilityOff } from '@mui/icons-material';
+import { useNavigate, Link as RouterLink } from 'react-router-dom';
+import GoogleIcon from '@mui/icons-material/Google';
+import GitHubIcon from '@mui/icons-material/GitHub';
+
+function SignUpScreen() {
+  const [username, setUsername] = useState('');
+  const [email, setEmail] = useState('');
```

Comment:
Adding the name of the account user would make it easier in the future to save the user information in the database